.circleci/config.yml

```diff
@@ -255,7 +255,7 @@ jobs:
       - run: php scripts/directory-yaml-parse.php
       # https://architecture.lullabot.com/adr/20211201-use-strict-types-in-php-code/
       # New php files must use strict types.
-      - run: scripts/phpcs-new-files
+      # - run: scripts/phpcs-new-files
```

Comment:
@arthurbaghdas Is this meant to be commented out?

Reply:
Yes, this was communicated with Moshe, and we decided to revisit this in a separate ticket which is already in a spec state